app/src/main/java/com/example/lhi06/residue/AlbumReview.java

```diff
@@ -1,32 +1,67 @@
 package com.example.lhi06.residue;
 
+import android.os.Parcel;
+import android.os.Parcelable;
 import android.support.annotation.NonNull;
 
 /**
  * Created by lhi06 on 20/07/2017.
  */
 
-public class AlbumReview {
+public class AlbumReview implements Parcelable{
```

Comment:
you should have a white space between `Parcelable` and `{`. Please reformat this class.